docs/en/tutorials/juicefs_on_colab.md

```diff
@@ -0,0 +1,151 @@
+massive high performance file storage system via `juicefs`. It is required that you have a Google Cloud Platform account
+to follow this demo document.
+
+Otherwise, if you have another cloud vendors' resources (for example AWS's RDBS and S3), you can still use this guide as
+a reference and with other reference docs provided by `juicefs` to achieve a similar solution.
+
+You might also want the Colab instance is in the same zone or close to the region where CloudSQL and GCS are deployed,
+to make JuiceFS reaching the best performance. The tutorial should work for a random hosted Colab instance but you might
+notice a slow performance due to the latency between the Colab instance and the CloudSQL/GCS regions. To start Colab
+instaces in a specific region,
+see [instructions for starting a GCE VM on Colab via GCP Marketplace](https://research.google.com/colaboratory/marketplace.html)
+.
+
+So in order to follow this guide you will need to have these resources ready:
+
+* A Google Cloud Platform account ready and also a *porject* created. In our case we will use GCP project
+  as `juicefs-learning` for the demo
+* A CloudSQL (Postgres) ready to be used. In this demo uses instance
+  `juicefs-learning:europe-west1:juicefs-sql-example-1` as the metadata service
+* A GCS bucket created as the object storage service. In this demo we use
+  `gs://juicefs-bucket-example-1` as the bucket to store file chunks.
+* An IAM ServiceAccount or an authrized user account that has write access to the posgtes server and GCS buckets
+
+## Detailed Steps
+
+### Step 1 - Format and mount a juicefs file system folder
+
+This step is only need to be done once, and you can choose to do this at any machine/instance where you have good
+connection and access to your Google Cloud resources.
+
+In this example I am doing this on my local machine. Firstly you can use
+`gcloud auth application-default login` to get a local credential ready or you can also
+use `GOOGLE_APPLICATION_CREDENTIALS` to setup as json key file.
+
+Then you can use [`cloud_sql_proxy`](https://cloud.google.com/sql/docs/mysql/connect-admin-proxy) to open a port (in
+this case 5432) locally so to expose your cloud Postgres service to your local machine:
+
+```
+gcloud auth application-default login
+
+# Or set up the json key file via GOOGLE_APPLICATION_CREDENTIALS=/path/to/key
+
+cloud_sql_proxy -instances=juicefs-learning:europe-west1:juicefs-sql-example-1=tcp:0.0.0.0:5432
+```
+
+Then use the following command to create a new file system with a name as `myvolume` by using `juicefs format` command.
+Then later we can mount this filesystem in any other machines/instances where you have access to your cloud resources.
+
+You can download `juicefs` [here](https://github.com/juicedata/juicefs/releases).
+
+```
+juicefs format \
+    --storage gs \
+    --bucket gs://juicefs-bucket-example-1 \
+    "postgres://postgres:mushroom1@localhost:5432/juicefs?sslmode=disable" \
```

Comment:
```suggestion
    "postgres://postgres:mushroom1@localhost:5432/juicefs?sslmode=disable" \
    # juicefs on the above line is the name of the database created in the Google SQL instance juicefs-sql-example-1 
```
Add an annotation here to explain what `juicefs` means in line 92. Please check if it is correct.